styles/dialog.css

```diff
@@ -66,9 +66,6 @@ ai-dialog > ai-dialog-header > button {
   padding: 0;
   cursor: pointer;
 }
-ai-dialog > ai-dialog-header > button:focus {
```

Comment:
I think that removing that whole commit is fine, it will be re-generated on release.